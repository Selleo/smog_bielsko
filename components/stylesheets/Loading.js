import StyleSheet from 'react-native-extensible-stylesheet'

export default StyleSheet.create({
  loadingContainer: {
    backgroundColor: 'white',
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'space-around'
  },
  loadingText: {
    fontSize: 18,
    textAlign: 'center'
  }
});
