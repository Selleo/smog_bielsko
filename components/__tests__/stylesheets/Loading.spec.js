import 'react-native';
import React                          from 'react';

import { shallow }                    from 'enzyme';
import chai, { expect }               from 'chai';
import dirtyChai                      from 'dirty-chai';

import Loading                         from '../../stylesheets/Loading';

const originalExpect = global.expect;

chai.use(dirtyChai);

describe('Loading', () => {
  it('renders correctly', () => {
    originalExpect(Loading).toMatchSnapshot();
    expect(Loading).to.exist();
  });
});
