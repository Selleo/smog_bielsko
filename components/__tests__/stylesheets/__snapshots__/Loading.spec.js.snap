exports[`Loading renders correctly 1`] = `
ExtensibleStyleSheet {
  "_options": Object {
    "loadingContainer": Object {
      "backgroundColor": "white",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "space-around",
    },
    "loadingText": Object {
      "fontSize": 18,
      "textAlign": "center",
    },
  },
  "_styles": Object {
    "loadingContainer": Object {
      "backgroundColor": "white",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "space-around",
    },
    "loadingText": Object {
      "fontSize": 18,
      "textAlign": "center",
    },
  },
  "_variables": Object {},
  "loadingContainer": Object {
    "backgroundColor": "white",
    "flex": 1,
    "flexDirection": "column",
    "justifyContent": "space-around",
  },
  "loadingText": Object {
    "fontSize": 18,
    "textAlign": "center",
  },
}
`;
